spec/controllers/miq_report_controller/dashboards_spec.rb

```diff
@@ -117,7 +117,7 @@
 
     describe "#db_save_members" do
       let(:set_data) do
-        Array.new(3) { |n| ["col#{(n + 1)}".to_sym, Array.new(2, FactoryBot.create(:miq_widget).id)] }.to_h
+        Array.new(2) { |n| ["col#{n + 1}".to_sym, Array.new(2, FactoryBot.create(:miq_widget).id)] }.to_h
```

Comment:
Think this is a lot easier to read and figure out what's going on. (not necessarily for this PR)

```suggestion
        {
          :col1 => 2.times.map { FactoryBot.create(:miq_widget).id },
          :col2 => 2.times.map { FactoryBot.create(:miq_widget).id }
        }
```

Reply:
it throwed an offence on Line 121 and 122 - Performance/TimesMap - Use **Array.new(2)** with a block instead of **.times.map**. Hence, used the recommended one..

Reply:
Interesting...it's more of a readability thing...surprised that rubocop likes the ugly one 😉